docs/kola.md

```diff
@@ -236,3 +236,10 @@ In order to see the logs for these tests you must enter the `tmp/kola/name_of_th
 
 `cosa run -i ignition_path` You can run it passing your Ignition, or the Ignition used in the the test that failed for troubleshooting reasons.
 
+## Run test on cloud
+`cosa kola run -p aws --aws-type t3.micro --aws-ami ami-0431766f2498820b8 --aws-region us-east-1 basic` This will run the basic tests on AWS using `ami-0431766f2498820b8` (fedora-coreos-37.20230227.20.2), how to create the credentials refer to https://github.com/coreos/coreos-assembler/blob/main/docs/mantle/credentials.md#aws
+
+`kola run -p=gce --gce-image=projects/fedora-coreos-cloud/global/images/fedora-coreos-37-20230227-20-2-gcp-x86-64 --gce-json-key=/data/gce.json --gce-project=fedora-coreos-testing basic` This will run the basic tests on GCP. 
+- `gce-image` is in the format of `projects/<GCP Image Project>/global/images/<GCP Image Name>`, to find related info refer to https://builds.coreos.fedoraproject.org/browser?stream=testing-devel&arch=x86_64.
+- `gce-json-key` is using a service account's JSON key for authentication, how to create service account keys refer to https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console.
```

Comment:
Hmm. We should probably link here to https://github.com/coreos/coreos-assembler/blob/main/docs/mantle/credentials.md#gce but that is definitely out of date.